src/System.Text.Encodings.Web/src/project.lock.json

```diff
@@ -1,18 +1,15 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
Yeah, all of our projects right now have them checked in. I think we should move away from that myself, but right now that's how we're doing it, in this repo at least. Since none of the dependencies are non-specific, I don't think it technically matters, although I'm not familiar enough with how NuGet works to be sure about that.